requirements.txt

```diff
@@ -1,12 +1,15 @@
-msprime=0.7.3
+msprime
 scikit-allel
 snakemake
 setuptools
 matplotlib
-pandas=0.24.2
```

Comment:
@andrewkern have you tested this with pandas versions more recent than 0.24.2? i was having problems with the masking that i found were due to some updates after 0.24.2. 

similarly, i also had some problems getting smc++ to run due to some changes to attrs after 19.1.0

Reply:
so yes i've tested this with the fancy new install instructions I give on the readme here https://github.com/popgensims/analysis/blob/master/n_t/readme.md

Works fine with this subtle change around.

Reply:
hmm maybe i'm missing something but this readme looks no different than the one thats been there for a while?